pkg/option/config.go

```diff
@@ -1448,6 +1451,9 @@ type DaemonConfig struct {
 	// AgentHealthPort is the TCP port for agent health status API
 	AgentHealthPort int
 
+	// ClusterHealthAddrs specifies IPs (v4/v6) for cluster-wide network connectivity health API to listen on (by default binds on all ifaces)
+	ClusterHealthAddrs []net.IP
```

Comment:
Could you use `netip.Addr` here? I know most fields in the package still use `net.IP`, but we're trying to migrate one change at a time.

Reply:
Ok, I can try to modify it to using `netip.Addr`.

Reply:
@gandro, it's using `netip.Addr` now. Please take a look again. Thanks.